### ISSUE TITLE
Stack trace obfuscated

### ISSUE THREAD

**Issue by abdimussa87** [getsentry/sentry-dart#1560]
### Platform

Flutter Mobile

### Obfuscation

Enabled

### Debug Info

Enabled

### Doctor

[✓] Flutter (Channel stable, 3.7.0, on macOS 13.4.1 22F770820d darwin-arm64
    (Rosetta), locale en-ET)
    • Flutter version 3.7.0 on channel stable at /Users/abdi/Development/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision b06b8b2710 (6 months ago), 2023-01-23 16:55:55 -0800
    • Engine revision b24591ed32
    • Dart version 2.19.0
    • DevTools version 2.20.1

[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.0)
    • Android SDK at /Users/abdi/Library/Android/sdk
    • Platform android-33, build-tools 33.0.0
    • ANDROID_HOME = /Users/abdi/Library/Android/sdk
    • Java binary at: /Applications/Android
      Studio.app/Contents/jre/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build
      11.0.12+0-b1504.28-7817840)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 14.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Build 14B47b
    • CocoaPods version 1.12.0

[✓] Android Studio (version 2021.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build
      11.0.12+0-b1504.28-7817840)

[✓] VS Code (version 1.80.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.68.0

[!] Connected device
    ! No devices available

[✓] HTTP Host Availability
    • All required HTTP hosts are available

### Version

7.8.0

### Steps to Reproduce

I just made an app bundle with this command:
`flutter build appbundle --obfuscate --split-debug-info=/path/to/obfuscation`

then uploaded the generated symbols to sentry using this command:
`sentry-cli upload-dif --org org-name --project project-name /path/to/obfuscation`

### Expected Result

Stack Trace isn't obfuscated

### Actual Result

Stack Trace is obfuscated

<img width="836" alt="Screenshot 2023-07-19 at 10 31 07 PM" src="https://github.com/getsentry/sentry-dart/assets/43067015/869da270-d135-48fc-864f-97ec28f79e1d">

### Are you willing to submit a PR?

None

**Comment by marandaneto** [comment]
@abdimussa87 Thanks, Android or iOS?
Can you try using the plugin? https://github.com/getsentry/sentry-dart-plugin
Also, take a look at https://docs.sentry.io/platforms/flutter/upload-debug/ just in case you've missed something.
Can you share the set of commands you have used to install the `appbundle` locally and test it on an emulator or something?

**Comment by abdimussa87** [comment]
@marandaneto thanks for responding. I didn't use the sentry-dart-plugin. I went ahead with uploading it manually. This is how it went.
<img width="563" alt="Screenshot 2023-07-20 at 7 41 02 PM" src="https://github.com/getsentry/sentry-dart/assets/43067015/db2d38d7-0aee-4e39-b93a-235f0be29be5">

The appbundle is uploaded to the playstore. I just saw some errors being reported on sentry, and the stack trace was obfuscated, even though I uploaded the symbols
